src/client/src/containers/About/index.tsx

```diff
@@ -35,21 +39,28 @@ const About = ({ versions, intl }: Props) => {
   const { formatMessage } = intl;
   return (
     <div className={styles.container}>
-      <div className={styles.title}>{formatMessage(messages.about)}</div>
       <p className={styles.repo}>
         <a
           className={styles.link}
           target="_blank"
           href={WEB_TEMPLATE_STUDIO_LINKS.REPO}
           onKeyUp={keyUpHandler}
         >
-          Web Template Studio
+          {formatMessage(messages.visitRepo)}
+        </a>
+        <a
+          className={styles.link}
+          href="https://github.com/Microsoft/WebTemplateStudio/issues"
```

Comment:
Can we have `target="_blank"` here. It won't matter on the extension but it is more intuitive to test on browser.

Reply:
Okay, I added it :)